docs/Advanced-Usage/Workloads/Booting-Fedora.rst

```diff
@@ -1,38 +1,56 @@
-.. _booting-fedora:
-
 Running Fedora on FireSim
-===========================
+=====================================
+All workload-generation related commands and code are in ``firesim/sw/firesim-software``.
 
-You can boot Fedora disk images pulled from upstream on FireSim simulations.
-These instructions assume you've already run through the tutorials.
+FireMarshal comes with a Fedora-based workload that you can use right out of
+the box in ``workloads/fedora-base.json``. We begin by building the
+workload (filesystem and boot-binary):
 
+::
 
-Building a FireSim-compatible Fedora Image
------------------------------------------------
+  ./marshal build workloads/fedora-base.json
 
-To download and build a Fedora-based Linux distro for FireSim, do the following:
+The first time you build a workload may take a long time (we need to download
+and decompress a pre-built fedora image), but subsequent builds of the same
+base will use cached results. Once the command completes, you should see two
+new files in ``images/``: ``fedora-base-bin`` and ``fedora-base.img``.  These
+are the boot-binary (linux + boot loader) and root filesystem (respectively).
+We can now launch this workload in qemu:
 
 ::
 
-    cd firesim/sw/firesim-software
-    ./sw-manager.py -c fedora-disk.json build
+  ./marshal launch workloads/fedora-base.json
```

Comment:
why not firemarshal?

Reply:
too many characters. waste-not, want-not. Also, I feel like firemarshal is too similar to firesim and I'd keep accidentally typing one when I meant the other.